doc/source/installation.rst

```diff
@@ -53,7 +53,16 @@ Install Molecule:
 
 .. code-block:: bash
 
-    $ sudo pip install molecule
+    $ pip install --user molecule
+
+Installing molecule package also installed its main script ``molecule``,
+usually in ``PATH``. Users should know that molecule can also be called as a
+python module, using ``python -m molecule ...``. This alternative method has
+some benefits:
+
+* allows to control which python interpreter is used by molecule
+* allows molecule installation at user level without even needing to have
```

Comment:
I'm not sure how this is related to the change.

Reply:
This clearly related to this patch because before it, if you would have installed molecule using --user it was likely that you would fail to execute "moleculte" because ~/.local/bin was not in you PATH. Addition of ~/.local/bin to user PATH is something relatively new.

In fact this is one of the main selling points for using module calling instead of script callling, as it does not depend of having the script in PATH.

Reply:
Oh.. I see. I'd say it's not _directly_ connected, but okay. It probably makes sense to leave this note here as well :)